.travis.yml

```diff
@@ -9,18 +9,33 @@ before_install:
     - curl -o- -L https://yarnpkg.com/install.sh | bash
     - export PATH=$HOME/.yarn/bin:$PATH
 install:
-    - yarn global add generator-jhipster yo
+    - yarn global add yo
     - yarn install
     - yarn link
 script:
+    # ----- local tests
     - yarn test
+
+    # ----- integration tests
+    # clone generator-jhipster
+    - cd $HOME
+    - git clone https://github.com/pascalgrimaud/generator-jhipster generator-jhipster
```

Comment:
once my PR https://github.com/jhipster/generator-jhipster/pull/7887 is merged, this line needs to be changed :
pascalgrimaud -> jhipster